HW_8/pacage/Task1.py

```diff
@@ -0,0 +1,21 @@
+# Напишите функцию, которая преобразует pickle файл хранящий список словарей в
+# табличный csv файл. Для тестированию возьмите pickle версию файла из предыдущей задачи.
+# Функция должна извлекать ключи словаря для заголовков столбца из переданного файла.
+
+__all__ = ['transformation']
+
+import pickle
+import csv
+
+def transformation(pickle_file, csv_file):
+    with(open(pickle_file, 'rb') as f_p,
+         open(csv_file, 'w', encoding='utf-8', newline='') as f_c):
+        dictionary = pickle.load(f_p)
+        new_csv = csv.DictWriter(f_c, fieldnames=['id', 'level', 'name'],
```

Comment:
Как я понял у вас список список словаре, тогда можно сделать так
```suggestion
        dictionary = pickle.load(f_p)
        new_csv = csv.DictWriter(f_c, fieldnames=dictionary[0],
```